layouts/tech-partner-form/single.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+  <html lang="en">
+    <head>
+      {{ partial "head" . }}
+    </head>
+  <body>
+      {{ partial "partner-form/form" .}}
+    <footer>
+      {{partial "footer" . }}
+    </footer>
+      {{ partial "site-scripts" .}}
+  </body>
+</html>
+
+<script type="text/javascript">
+    window.onload = function() {
+
+    var nextButton = document.getElementById("partner-form-next");
+    var backButton = document.getElementById("partner-form-back");
+    var submitButton = document.getElementById("partner-form-submit");
+    var parent = document.getElementsByClassName("step"); // all step wrapper divs
+    var progressBar = document.getElementById("progress-bar");
+    var errElm = document.getElementsByClassName("error"); // all error divs
+    var formStart = document.getElementById("start-wrapper"); // initial page message
+    var startButton = document.getElementById("partner-form-start");
+    var firstStep = document.getElementById("partner-form-name");
+    var progressBarWrapper = document.getElementById("progress-bar-wrapper");
+    var stepButtons = document.getElementById("step-buttons");
+
+
+    var index = 0; // to be increased an decreased depending on button clicked
+    var progressBarValue = 8.333; // 100 divided by the 12 steps
+
+    var partner_form = {partner_form: true}
+
+    if(formStart && startButton && firstStep && progressBarWrapper && stepButtons) {
+      // remove beginning message and start process
+      startButton.onclick = function (event) {
+        formStart.setAttribute("style", "display:none;");
+        firstStep.setAttribute("style", "display:inherit;");
+        progressBarWrapper.setAttribute("style", "display:inherit;");
+        stepButtons.setAttribute("style", "display:inherit;");
+      }
+    }
+
+    nextButton.onclick = function (event) {
+      // parent[index] will always be current step
+      if(parent[index] && errElm && errElm[index]) {
+        // if required field is empty
+        if(parent[index].querySelector(".required") && parent[index].querySelector(".required").value == "") {
+          errElm[index].setAttribute("style", "display:block;");
+        } else {
+          var userEmailInput = parent[index].querySelector(".user-email");
+          // if email input does not include ampersand
+          if(userEmailInput && userEmailInput.value && !userEmailInput.value.includes("@")) {
```

Comment:
i guess there is a reason to use regular expressions here.

At least, we have to check if there is '.' and userEmailInput.length > 3